src/components/PlaceList.tsx

```diff
@@ -17,8 +17,15 @@ const PlaceList: React.FC<PlaceListProps> = ({ places, filters }) => {
       const { startDate, endDate } = dateRange
 
       for (let book of placeData.bookings) {
-        if (startDate.getTime() <= book.lastDay && endDate.getTime() >= book.firstDay) {
-          return false
+        if (book.endDate) {
+          if (
+            endDate >= book.startDate.toDate() &&
+            startDate <= book.endDate.toDate()
+          ) {
+            return false
+          } else if (endDate > book.startDate.toDate()) {
```

Comment:
No need for `else` here. Just start with `if` on the new line

Reply:
Also, the second expression is more simple, so it makes sense to run it the first